### PR TITLE
Fix add repo command and wrong link

### PR DESCRIPTION
**Pull request by raquel-campuzano**
The command for adding the stable helm charts repo should be fixed in all README files.

**Description of the change**

Replace the add repo command to use the helm stable repo instead of the bitnami one. Fix the link to the Kubernetes documentation about persistent volumes. 

**Benefits**

Helm 3 users can install the chart repository in their clusters.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

The replacing of the helm repo add command should be done in all READMEs for all the charts. 

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mysql/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## TL;DR;
 
 ```bash
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com
 $ helm install my-release bitnami/mysql
```
Comment (carrodher) [inline bitnami/mysql/README.md]: mysql is not in the _stable_ repo so those instructions doesn't work:
```
▶ helm repo list
Error: no repositories to show

▶ helm repo add stable https://kubernetes-charts.storage.googleapis.com
”stable” has been added to your repositories

▶ helm install my-release bitnami/mysql
Error: failed to download “bitnami/mysql” (hint: running `helm repo update` may help)
```

Adding the _bitnami_ repo it works as expected:
```
▶ helm repo add bitnami https://charts.bitnami.com/bitnami
“bitnami” has been added to your repositories

▶ helm install my-release bitnami/mysql
NAME: my-release
LAST DEPLOYED: Mon Feb 17 16:08:10 2020
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Please be patient while the chart is being deployed

Tip:

  Watch the deployment status using the command: kubectl get pods -w --namespace default

Services:

  echo Master: my-release-mysql.default.svc.cluster.local:3306
  echo Slave:  my-release-mysql-slave.default.svc.cluster.local:3306

Administrator credentials:

  echo Username: root
  echo Password : $(kubectl get secret --namespace default my-release-mysql -o jsonpath=“{.data.mysql-root-password}” | base64 --decode)

To connect to your database:

  1. Run a pod that you can use as a client:

      kubectl run my-release-mysql-client --rm --tty -i --restart=‘Never’ --image  docker.io/bitnami/mysql:8.0.19-debian-10-r0 --namespace default --command -- bash

  2. To connect to master service (read/write):

      mysql -h my-release-mysql.default.svc.cluster.local -uroot -p my_database

  3. To connect to slave service (read-only):

      mysql -h my-release-mysql-slave.default.svc.cluster.local -uroot -p my_database

To upgrade this helm chart:

  1. Obtain the password as described on the ‘Administrator credentials’ section and set the ‘root.password’ parameter as shown below:

      ROOT_PASSWORD=$(kubectl get secret --namespace default my-release-mysql -o jsonpath=“{.data.mysql-root-password}” | base64 --decode)
      helm upgrade my-release bitnami/mysql --set root.password=$ROOT_PASSWORD
```

File: bitnami/mysql/README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com
```
Comment (carrodher) [inline bitnami/mysql/README.md]: Adding this change
```suggestion
$ helm repo add bitnami https://charts.bitnami.com/bitnami
```
and bumping the chart version in the `bitnami/mysql/Chart.yaml` file is enough to merge this PR.
